### PR TITLE
Peer code review PR 🧠

### PR DESCRIPTION
**Pull request by aalo826**
review yes

### CODE REVIEW

File: src/Components/Sprays.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+  useEffect(() => {
+    axios({
+      url: "https://valorant-api.com/v1/sprays",
+      method: "GET",
+      dataRespone: "json",
+    }).then((response) => {
+      const sprayArray = response.data.data;
+
+      const finalData = sprayArray.map((sprayData, index) => {
```
Comment (victoria-reis) [inline src/Components/Sprays.js:16]: so satisfying to see your data so organized!

File: src/Components/Sprays.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            display:
+              "https://media.valorant-api.com/sprays/25d82d17-4739-e206-e8aa-d1bd4fd4dae6/fulltransparenticon.png",
+            id: "25d82d17-4739-e206-e8aa-d1bd4fd4dae6",
+            key: 42,
+          };
+        }
+
+      });
+      setSpray(finalData);
+      console.log(finalData)
```
Comment (victoria-reis) [inline src/Components/Sprays.js:49]: don't forget to delete this console log before pushing to production

**Review by victoria-reis — APPROVED** [review]
Hey, your code is super clean and organized! You should be proud. After looking at the deployed project I also really like the design, great job.

File: src/Components/Sprays.js (hunk 2 above)
Comment (aalo826) [reply src/Components/Sprays.js:49]: thanks for the reminder

**Pull request merged**